src/hooks/shapediver/viewer/useDrawingTools.ts

```diff
@@ -0,0 +1,45 @@
+import { useEffect } from "react";
+import { Callbacks, CustomizationProperties, createDrawingTools } from "@shapediver/viewer.features.drawing-tools";
+import { useShapeDiverStoreViewer } from "store/useShapeDiverStoreViewer";
+import { useDrawingToolsApiStore } from "store/useDrawingToolsApiStore";
+
+/**
+ * Hook allowing to create the drawing tools.
+ * 
+ * @param viewportId 
+ */
+export function useDrawingTools(viewportId: string, customizationProperties?: CustomizationProperties, callbacks?: Callbacks): void {
```

Comment:
@MajorMeerkatThe3rd it appears strange that this hook doesn't return the drawing tools API. You can omit using zustand here and use a `useRef<IDrawingToolsApi | undefined>` whose value you set from the useEffect hook.